packages/expo-av/src/Audio/Sound.ts

```diff
@@ -14,6 +14,40 @@ import { PitchCorrectionQuality } from '../Audio';
 import ExponentAV from '../ExponentAV';
 import { throwIfAudioIsDisabled } from './AudioAvailability';
 
+export interface AudioChannel {
```

Comment:
Use types instead of interfaces. Interfaces are more intended when you expect the user to write his own class that implements the interface.